include/nbl/builtin/hlsl/glsl_compat/subgroup_shuffle.hlsl

```diff
@@ -38,6 +38,18 @@ T subgroupShuffleDown(T value, uint delta)
 #endif
 }
 
+
+template<typename T>
+T subgroupShuffleXor(T value, uint32_t delta)
+{
+#ifdef NBL_GL_KHR_shader_subgroup_shuffle_relative
+    return spirv::groupShuffleXor<T>(3, value, delta);
```

Comment:
@kpentaris do we assume shuffle relative always there?

Reply:
It was not present in all devices if I remember correctly, which is why I added the #ifdef, but that might be the case for devices that are no longer supported by Nabla.

Reply:
we require it now https://github.com/Devsh-Graphics-Programming/Nabla/blob/56d4efb3fbbfca936f6cf8936e53b31823d4ad1d/src/nbl/video/vulkan/profiles/NablaCore.json#L328

Reply:
remove the ifdef (if its still there after merging `master` to this branch and resolving conflicts)